src/mapbox/mapbox.js

```diff
@@ -111,33 +157,47 @@ function checkPropTypes(props, component = 'component') {
 // - Provides support for specifying tokens during development
 
 export default class Mapbox {
-  constructor(props) {
+
+  static initialized : boolean = false;
+  static propTypes : any = propTypes;
+  static defaultProps : any = defaultProps;
+  static savedMap : any = null;
+
+  constructor(props: Props) {
     if (!props.mapboxgl) {
       throw new Error('Mapbox not available');
     }
 
+    this.mapboxgl = props.mapboxgl;
+
     if (!Mapbox.initialized && console.debug) { // eslint-disable-line
       Mapbox.initialized = true;
 
       // Version detection using babel plugin
       /* global __VERSION__ */
+      // $FlowFixMe
```

Comment:
If the error is that `__VERSION__` is undefined, just `declare var __VERSION__` somewhere (either in this file or a header file somewhere)

Reply:
Then lint is not happy (__VERSION__ defined but not used) because __VERSION__ is replaced with the actual version string by the babel plugin version-inline.

Reply:
Weird. Oh well!